lib/initialize.js

```diff
@@ -0,0 +1,98 @@
+const path = require('path')
+const rimraf = require('rimraf')
+const fs = require("fs")
+const constants = require("./constants")
+const {createCustomFileAccessor} = require("./config")
+const fileFuncs = require("./file-operations");
+
+//The two locations parsec app data folder might exist
+const parsecDataLocationDefault = path.join(constants.appDataPath,'..','Parsec')
+const parsecDataLocationOther = "C:\\ProgramData\\Parsec"
+
+//flag for parsecd.exe found
+
+
+//flag for parsec app data location found
+const flags = {
+    parsecdFound : 0,
+    parsecDataLocationFound : 0
+}
```

Comment:
y not make this a part of global_state, feels like it should be?

Reply:
done